libqtile/backend/x11/window.py

```diff
@@ -1044,14 +1044,15 @@ def change_layer(self, up=True, top_bottom=False):
         windows.sort(key=lambda w: stack.index(w[0].wid))
 
         # Get lists of windows on lower, higher or same "layer" as window
-        lower = [w[0].wid for w in windows if w[1] is None and w[2] > layering]
-        higher = [w[0].wid for w in windows if w[1] is None and w[2] < layering]
-        same = [w[0].wid for w in windows if w[1] is None and w[2] == layering]
```

Comment:
Is there any reason why we should exclude transient windows when calculating the list of windows in lower, same and higher layers? I can't think of any.

If we don't need to exclude them then I think we can remove the transient field from the windows tuple.

Reply:
The only thing that matters is they are above their parent, but that doesn't require excluding them here does it?